py/vtdb/vtgate_utils.py

```diff
@@ -117,6 +117,9 @@ def convert_to_dbexception(self, args):
     Returns:
       An exception from dbexceptions.
     """
+    # FIXME(alainjobart): this is extremely confusing: self.message is only
```

Comment:
Yeah, to be honest, I'm not sure why we handle integrity error message pruning differently from all other errors.